From 605c3bd42fbec1fa1c09e35bb93c6c577110b870 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Mon, 18 May 2020 12:56:27 +0200
Subject: [PATCH 212/220] tpm: Enable module for all EFI platforms

The tpm module is only enabled for x86_64, but there's nothing specific to
that architecture in the code and could be enabled for all EFI platforms.

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 grub-core/Makefile.core.def | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/Makefile.core.def b/grub-core/Makefile.core.def
index 6619946..b283c50 100644
--- a/grub-core/Makefile.core.def
+++ b/grub-core/Makefile.core.def
@@ -2512,7 +2512,7 @@ module = {
   name = tpm;
   common = commands/tpm.c;
   efi = commands/efi/tpm.c;
-  enable = x86_64_efi;
+  enable = efi;
 };
 
 module = {
-- 
1.8.3.1

